### PR TITLE
Add notifications to Keba EV chargers

### PR DESCRIPTION
**Pull request by matlimatli**
## Proposed change

Add documentation for new service that adds the possibility to display text messages on the display of Keba EV chargers. 

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: [https://github.com/home-assistant/core/pull/36056](https://github.com/home-assistant/core/pull/36056)
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/keba.markdown [hunk 1]
```diff
@@ -120,6 +120,17 @@ The service `keba.set_failsafe` sets the failsafe mode of the charging station.
 }
 ```
 
+### Show Text on Display `keba.set_text``
+
+The service `keba.set_text` shows the submitted text on the display of chargers with built-in LED display. `min_time` is the minimum time in seconds the text will be shown if another message is requested. `max_time` is the maximum time to display the message when nothing else is requested. A maximum of 23 characters can be shown. Horizontal scroll is performed autiomatically when needed. Payload example:
```
Comment (klaasnicolaas) [inline source/_integrations/keba.markdown]:
```suggestion
The service `keba.set_text` shows the submitted text on the display of chargers with built-in LED display. `min_time` is the minimum time in seconds the text will be shown if another message is requested. `max_time` is the maximum time to display the message when nothing else is requested. A maximum of 23 characters can be shown. Horizontal scroll is performed autiomatically when needed. Payload example:

```

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

File: source/_integrations/keba.markdown [hunk 2]
```diff
@@ -120,6 +121,45 @@ The service `keba.set_failsafe` sets the failsafe mode of the charging station.
...
+
+```json
+{
+  "message": "Welcome home!",
+  "data": {
+    "min_time": 4,
+    "max_time": 20
+  }
+}
+```
```
Comment (frenck) [inline source/_integrations/keba.markdown]: We use YAML in the frontend for this nowadays.
Please convert this example to YAML.

File: source/_integrations/keba.markdown [hunk 3]
```diff
@@ -120,6 +121,45 @@ The service `keba.set_failsafe` sets the failsafe mode of the charging station.
...
+  type: string
+{% endconfiguration %}
+
+### Usage
+
+The use of the notify service is [described here](/integrations/notify/).
+
+The `message` part of the event payload is shown on the display. Scrolling is performed if needed. A maximum of 23 characters can be shown.
+
+The optional `data` part may contain specifications of the message duration. `min_time` is the minimum time in seconds the text will be shown if another message is requested. `max_time` is the maximum time to display the message when nothing else is requested. By default the message is shown minimum 2 seconds and maximum 10 seconds.
```
Comment (frenck) [inline source/_integrations/keba.markdown]:
```suggestion
The use of the notify service is [described here](/integrations/notify/).

The `message` part of the event payload is shown on the display. Scrolling is performed if needed. A maximum of 23 characters can be shown.

The optional `data` part may contain specifications of the message duration. `min_time` is the minimum time in seconds the text will be shown if another message is requested. `max_time` is the maximum time to display the message when nothing else is requested. By default, the message is shown a minimum of 2 seconds and a maximum of 10 seconds.
```

File: source/_integrations/keba.markdown [hunk 4]
```diff
@@ -18,7 +19,7 @@ This component provides the following platforms:
 - Binary Sensors: Online state, plug state, Charging state and failsafe mode state.
 - Lock: Authorization (like with the RFID card).
 - Sensors: current set by the user, target energy set by the user, charging power, charged energy of the current session and total energy charged.
-- Services: authorize, deauthorize, set energy target, set maximum allowed current and manually update the states. More details can be found [here](/integrations/keba/#services).
+- Services: authorize, deauthorize, set energy target, set maximum allowed current, show text and manually update the states. More details can be found [here](/integrations/keba/#services).
```
Comment (frenck) [inline source/_integrations/keba.markdown]:
```suggestion
- Services: authorize, deauthorize, set energy target, set the maximum allowed current, show text and manually update the states. More details can be found [here](/integrations/keba/#services).
```

**Review by frenck — CHANGES_REQUESTED** [review]

**Review by frenck — APPROVED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.

**Pull request merged**